Sentinel/Classes/TextEditing/TextEditingTool.swift

```diff
@@ -5,19 +5,23 @@
 //  Created by Vlaho Poluta on 31/07/2020.
 //
 
-import UIKit
+import SwiftUI
 
 @objcMembers
-public class TextEditingTool: NSObject, Tool {
-    
+public final class TextEditingTool: NSObject, Tool {
```

Comment:
There is no need for subclassing the `NSObject` class anymore - this can be defined as a struct.

_Note: check other usages of `NSObject` and update if necessary._